frontend/.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "plugins": ["react", "react-hooks"],
+  "parser": "babel-eslint",
+  "extends": ["eslint:recommended", "plugin:react/recommended", "plugin:prettier/recommended"],
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2018
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "linebreak-style": ["error", "unix"],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+    "react/prop-types": 0
```

Comment:
add few more rules :
- proptypes’y w componentach alfabetycznie
```javascript
    "react/sort-prop-types": [2, {
      "callbacksLast": false,
      "ignoreCase": true,
      "requiredFirst": false,
      "sortShapeProp": true
    }],
```
- require trailing commas on multilines
`'comma-dangle': [2, 'always-multiline'],`
- show if debugger was left
`'no-debugger': 2,`
- Require === and !== 
`'eqeqeq': 2,`
- enforce the consistent use of either double or single quotes in JSX attributes
`"jsx-quotes": [2, "prefer-single"],`
- enforce consistent spacing before and after keywords
`'keyword-spacing': 2,`

Reply:
after call don't use
- enforce the consistent use of either double or single quotes in JSX attributes
"jsx-quotes": [2, "prefer-single"],